packages/utils/interactive-hooks/src/useKeyProps.win32.ts

```diff
@@ -37,20 +37,26 @@ export function getKeyCallbackWorker(userCallback?: KeyCallback, ...keys: string
 function getKeyUpPropsWorker(userCallback: KeyCallback, ...keys: string[]): KeyPressProps {
   return {
     onKeyUp: getKeyCallbackWorker(userCallback, ...keys),
+    keyUpEvents: keys.map((keyCode) => {
+      return { key: keyCode };
+    }),
```

Comment:
suggestion: maybe add a comment on what this is doing/why?

Reply:
I guess there's a comment in Submenu so maybe it's unnecessary

Reply:
It's turning `['a', 'b', 'c']` into `[{key: 'a'}, {key: 'b'}, {key: 'c}]`. I figured that was simple enough that it didn't need a comment.

The reason for the array of objects is to support key combinations. You might want Ctrl+Enter (`{ ctrlKey: true, key: 'Enter'}`) to be suppressed, while Shift+Enter passes through.

Reply:
Oh I meant the general intent of supplying the events to the prop so that they don't get listened to on the native side, not the array mapping

Reply:
Ah. That would be a super simple one liner that avoids needing to find this PR and read my paragraphs of PR notes. Nice catch!